config/spotbugs-exclude.xml

```diff
@@ -404,4 +404,24 @@
     <Method name="getTargetFiles" />
     <Bug pattern="RCN_REDUNDANT_NULLCHECK_WOULD_HAVE_BEEN_A_NPE" />
   </Match>
+
+  <!-- Suppress NP_NULL_ON_SOME_PATH_FROM_RETURN_VALUE -->
+  <!-- until https://github.com/checkstyle/checkstyle/issues/16979 -->
```

Comment:
This not same problem..
Please share link to violations or just copy paste here violations from log